src/components/AvatarCropModal/AvatarCropModal.js

```diff
@@ -270,10 +270,14 @@ const AvatarCropModal = (props) => {
             height: size, width: size, originX, originY,
         };
 
+        const isSvg = props.imageType.includes('image/svg');
+        const imageName = isSvg ? 'fileName.png' : props.imageName;
+        const imageType = isSvg ? 'image/png' : props.imageType;
```

Comment:
I believe we should some comments here, on why  thoughts @Beamanator

Reply:
Looks good, please add it!

Reply:
Looks good for me, @Beamanator please check if this is fine!

Reply:
I like the new comment, but what do y'all think about explaining a bit more _why_ we're doing this conversion / _why_ we're specifically choosing png? I think if we add that and everything is perfectly clear we'll be good to go 👍

Reply:
Hmm sorry to be really picky, what I was thinking to explain the reason we choose "png" is just to preserve SVG transparency, right? So what about something like:

```
// Svg images are converted to a png blob to preserve transparency, so we need to update the
// image name and type accordingly.
```